include/k4a/k4a.h

```diff
@@ -886,13 +947,76 @@ K4A_EXPORT uint32_t k4a_image_get_white_balance(k4a_image_t image_handle);
  */
 K4A_EXPORT uint32_t k4a_image_get_iso_speed(k4a_image_t image_handle);
 
-/** Set the time stamp, in microseconds, of the image.
+/** Set the device time stamp, in microseconds, of the image.
+ *
+ * \param image_handle
+ * Handle of the image to set the timestamp on.
+ *
+ * \param timestamp_usec
+ * Device timestamp of the image in microseconds.
+ *
+ * \remarks
+ * Use this function in conjunction with k4a_image_create() or k4a_image_create_from_buffer() to construct a
+ * \ref k4a_image_t.
+ *
+ * \remarks
+ * Set a timestamp of 0 to indicate that the timestamp is not valid.
+ *
+ * \remarks
+ * The device timestamp represents the mid-point of exposure of the image, as captured by the hardware.
+ *
+ * \relates k4a_image_t
+ *
+ * \xmlonly
+ * <requirements>
+ *   <requirement name="Header">k4a.h (include k4a/k4a.h)</requirement>
+ *   <requirement name="Library">k4a.lib</requirement>
+ *   <requirement name="DLL">k4a.dll</requirement>
+ * </requirements>
+ * \endxmlonly
+ */
+K4A_EXPORT void k4a_image_set_device_timestamp_usec(k4a_image_t image_handle, uint64_t timestamp_usec);
+
+/** Set the device time stamp, in microseconds, of the image.
  *
  * \param image_handle
  * Handle of the image to set the timestamp on.
  *
  * \param timestamp_usec
- * Timestamp of the image in microseconds.
+ * Device timestamp of the image in microseconds.
+ *
+ * \remarks
+ * Use this function in conjunction with k4a_image_create() or k4a_image_create_from_buffer() to construct a
+ * \ref k4a_image_t.
+ *
+ * \remarks
+ * Set a timestamp of 0 to indicate that the timestamp is not valid.
+ *
+ * \remarks
+ * The device timestamp represents the mid-point of exposure of the image, as captured by the hardware.
+ *
+ * \relates k4a_image_t
+ *
+ * \deprecated
+ * Deprecated starting in 1.1.0. Please use k4a_image_set_device_timestamp_usec().
```

Comment:
Just verifying this should still say deprecated in 1.1.0?

Reply:
Nope, good catch